toolip/utils/time.py

```diff
@@ -0,0 +1,29 @@
+from datetime import datetime, timezone
+
+import pytz
+
+
+def now():
+    return datetime.now(timezone.utc)
+
+
+def now_epoch() -> int:
+    return int(datetime.now(timezone.utc).timestamp())
+
+
+def now_epoch_ms() -> int:
+    return int(datetime.now(timezone.utc).timestamp() * 1000)
+
+
+def make_time_aware(dtime: datetime):
+    return dtime.replace(tzinfo=pytz.utc)
+
+
+def sh_to_dt_time(shtime):
+    if ':' == shtime[-3:-2]:
+        shtime = shtime[:-3] + shtime[-2:]
+    try:
+        result = datetime.strptime(shtime, '%Y-%m-%dT%H:%M:%S%z')
+    except Exception:
+        result = shtime
+    return result
```

Comment:
That's Shopify specific so it belongs to SPyLib, and it should be tested for more recent versions of datetime ... maybe it can parse Shopify's format without this weird string manipulation.
Also this needs testing.